### PR TITLE
fix: add SourceContext to fix storybook

### PR DESCRIPTION
**Pull request by pieterlukasse**
Jira Ticket: NA

### Bug Fixes
- fix: added SourceContext to fix storybook

### CODE REVIEW

File: src/Analysis/GWASV2/Shared/SourceMock.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+// ======== MOCK SourceContext for storybooks and tests ================ //
+
+const SourceMockContext = createContext();
+
+export function SourceMockContextProvider({ children }) {
+  const [source, setSource] = useState(undefined);
+  const sourceId = 123; // hard-coded MOCK value instead of fetching from cohort-middleware
+  useEffect(() => {
+    setSource(sourceId);
```
Comment (jarvisraymond-uchicago) [inline src/Analysis/GWASV2/Shared/SourceMock.jsx:14]: Could we just set the state to have a default value of 123 at line 11? Or am I missing something?

**Comment by pieterlukasse** [comment]
closing this PR as it does not really solve the issue....

**Comment by pieterlukasse** [comment]
reopening for a different solution

File: src/Analysis/GWASV2/Shared/SourceMock.jsx (hunk 1 above)
Comment (jarvisraymond-uchicago) [reply src/Analysis/GWASV2/Shared/SourceMock.jsx]: Never mind, no longer relevant.

**Review by jarvisraymond-uchicago — APPROVED** [review]
Real good! Easy solution!

**Pull request merged**